### PR TITLE
Cloning without records admin toggle

### PR DESCRIPTION
**Pull request by Luxizzle**
## About the PR 

This PR adds a toggle in the **Server Toggles** tab, allowing an admin to switch off cloning records completely. After this, the crew is left with a cloning terminal that has no more "Records" tab and when the scanner has an occupant, the "Scan" button is replaced with a "Scan & Clone" button, which will immediately clone a dead crewmate and refusing alive occupants.

Cloning from disks for security members is still available, except instead of loading the record, the option will now be to clone straight from the disk.

## Why's this needed? 

Cloning is very integral to the stations' lifecycle, and it has come up quite a few times that maybe cloning is too easy and death has no meaning due to how easy it is to clone another crew member by the click of a button. It has always been this way, so at least in my opinion, it is also hard to judge a station with something different.

Whenever a cloner gets missing, we immediately feel this whenever we want to go and clone somebody, so removing the cloner completely feels like a very drastic choice, and one that I personally don't feel would be positive. So instead of removing the cloner, this gives us the option to experiment with a different style of cloning.

With disabling records, there is more importance in your body being found, preserved and brought to medbay to be cloned, health implants and Rotbusttec implants become more important. And hopefully the feeling that death has more meaning is strengthened.

I hope this will lead to more experimentation with the cloner and its surrounding methods of revival.

### CODE REVIEW

**Comment by CaioFrancisco** [comment]
> I like this idea, gives us a way to test this without it being fully in place. Just one question though, how does this affect security clone disks? Do they no longer function or immediately clone the officer?

Since Security members are incredibly important to the station, I think they should keep a one-use disk in case their body is found too late.

**Comment by Luxizzle** [comment]
> > I like this idea, gives us a way to test this without it being fully in place. Just one question though, how does this affect security clone disks? Do they no longer function or immediately clone the officer?
> 
> Since Security members are incredibly important to the station, I think they should keep a one-use disk in case their body is found too late.

Good point, added a "Clone from disk" option when this is turned on.

**Review by flrsh — APPROVED** [review]
Looks good to me, though I'm not too familiar with TGUI.

File: tgui/packages/tgui/interfaces/CloningConsole.js [hunk 1]
```diff
@@ -219,17 +227,27 @@ const Functions = (props, context) => {
...
+                </Button>
+              )}
+              {!cloningWithRecords && (
+                <Button
+                  icon="upload"
+                  color={"blue"}
+                  onClick={() => act("loadAndClone")}>
+                  Clone from disk
+                </Button>
+              )}
```
Comment (codeJester27) [inline tgui/packages/tgui/interfaces/CloningConsole.js:250]:
```suggestion
              {cloningWithRecords ? (
                <Button
                  icon="upload"
                  color={"blue"}
                  onClick={() => act("load")}>
                  Load from disk
                </Button>
              ) : (
                <Button
                  icon="upload"
                  color={"blue"}
                  onClick={() => act("loadAndClone")}>
                  Clone from disk
                </Button>
              )}
```

**Comment by ZeWaka** [comment]
!merge_upstream

**Comment by ZeWaka** [comment]
!merge_upstream

**Comment by ZeWaka** [comment]
!merge_upstream

**Pull request merged**